whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/WhoisSlaveDao.java

```diff
@@ -0,0 +1,32 @@
+package net.ripe.db.nrtm4.persist;
+
+import org.javatuples.Pair;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+
+@Service
+public class WhoisSlaveDao {
+
+    private final WhoisSlaveRepository whoisSlaveRepository;
+
+    public WhoisSlaveDao(
+        final WhoisSlaveRepository whoisSlaveRepository
+    ) {
+        this.whoisSlaveRepository = whoisSlaveRepository;
+    }
+
+    public List<Pair<SerialModel, RpslObjectModel>> findSerialsAndObjectsSinceSerial(final int serialId) {
+        final List<Pair<SerialModel, RpslObjectModel>> lastObjects = whoisSlaveRepository.findSerialsInLastSince(serialId);
+        final List<Pair<SerialModel, RpslObjectModel>> historyObjects = whoisSlaveRepository.findSerialsInHistorySince(serialId);
```

Comment:
we are already have a versionDao in whois-commons. Can we use it instead creating new?

Reply:
yes, i moved the code there